source/PlayniteSDK/Models/Game.cs

```diff
@@ -177,7 +177,9 @@ public enum GameField : int
         ///
         LastSizeScanDate = 90,
         ///
-        RecentActivity = 91
+        RecentActivity = 91,
+        ///
+        Hdr = 92

```

Comment:
Should be EnableSystemHdr since that's how the original property is named.

Reply:
Done